core/domain/platform_feature_services.py

```diff
@@ -160,30 +157,16 @@ def _get_server_mode():
         if constants.DEV_MODE else platform_parameter_domain.SERVER_MODES.prod)
 
 
-def _create_evaluation_context_for_server(user_settings):
+def _create_evaluation_context_for_server():
     """Returns evaluation context with information of the server.
 
-    Args:
-        user_settings: UserSettings|None. The settings of the current logged in
-            user, if any.
-
     Returns:
         EvaluationContext. The context for evaluation.
     """
-    language_code = 'en' # Default language code if no preference is set.
-    if (
-            user_settings is not None and
-            user_settings.preferred_site_language_code is not None):
-        language_code = user_settings.preferred_site_language_code
-
-    # TODO(#11073): Properly set app version below, and rename the user_locale
```

Comment:
Can you keep the part of this TODO corresponding to the version, but update it to point to the correct issue: #11208? It's still relevant & worth tracking.

Reply:
Done.